pkgs/development/interpreters/bqn/cbqn/default.nix

```diff
@@ -74,43 +66,40 @@ stdenv.mkDerivation rec {
   '' + (if genBytecode then ''
     ${bqn-path} ./build/genRuntime ${mbqn-source} build/bytecodeLocal/
   '' else ''
-    cp ${cbqn-bytecode-files}/src/gen/{compiles,explain,formatter,runtime0,runtime1,src} build/bytecodeLocal/gen/
-  '')
-  + lib.optionalString enableReplxx ''
+    cp ${cbqn-bytecode-submodule}/gen/{compiles,explain,formatter,runtime0,runtime1,src} build/bytecodeLocal/gen/
+  '') + lib.optionalString enableReplxx ''
     cp -r ${replxx-submodule} build/replxxLocal/
   '';
 
-  outputs = [
-    "out"
-  ] ++ lib.optionals enableLibcbqn [
-    "lib"
-    "dev"
-  ];
+  outputs = [ "out" ] ++ lib.optionals enableLibcbqn [ "lib" "dev" ];
 
   installPhase = ''
-     runHook preInstall
+    runHook preInstall
 
-     mkdir -p $out/bin/
-     cp BQN -t $out/bin/
-     # note guard condition for case-insensitive filesystems
-     [ -e $out/bin/bqn ] || ln -s $out/bin/BQN $out/bin/bqn
-     [ -e $out/bin/cbqn ] || ln -s $out/bin/BQN $out/bin/cbqn
-  ''
-  + lib.optionalString enableLibcbqn ''
-     install -Dm644 include/bqnffi.h -t "$dev/include"
-     install -Dm755 libcbqn${stdenv.hostPlatform.extensions.sharedLibrary} -t "$lib/lib"
-  ''
-  + ''
-     runHook postInstall
+    mkdir -p $out/bin/
+    cp BQN -t $out/bin/
+    # note guard condition for case-insensitive filesystems
+    [ -e $out/bin/bqn ] || ln -s $out/bin/BQN $out/bin/bqn
+    [ -e $out/bin/cbqn ] || ln -s $out/bin/BQN $out/bin/cbqn
+  '' + lib.optionalString enableLibcbqn ''
+    install -Dm644 include/bqnffi.h -t "$dev/include"
+    install -Dm755 libcbqn${stdenv.hostPlatform.extensions.sharedLibrary} -t "$lib/lib"
+  '' + ''
+    runHook postInstall
   '';
 
   meta = with lib; {
     homepage = "https://github.com/dzaima/CBQN/";
     description = "BQN implementation in C";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ AndersonTorres sternenseemann synthetica shnarazk ];
+    maintainers = with maintainers; [
```

Comment:
Why are you so inconsistent?

Reply:
I used `nixpkgs-fmt` to do the formatting.